.github/workflows/build.yml

```diff
@@ -0,0 +1,269 @@
+    needs: credential-check
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        architecture: [x86_64, aarch64]
+    env:
+      # This image is important. The default manylinux2014 images are problematic to configure for cross-compilation
+      # in a way that will satisfy the gmp configure script.
+      DOCKER_IMAGE: messense/manylinux2014-cross:${{ matrix.architecture }}
+      TARGET: ${{ matrix.architecture }}-unknown-linux-gnu
+      ARCH_LIB: libopendp-${{ matrix.architecture }}.so
+    steps:
+      - name: Checkout repository
+        if: ${{ !inputs.fake }}
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.channel }}
+
+      - name: Cache Rust dependencies
+        if: ${{ !inputs.fake }}
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin
+            ~/.cargo/registry/index
+            ~/.cargo/registry/cache
+            ~/.cargo/git/db
+            ~/.cache/gmp-mpfr-sys
+            rust/target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('rust/Cargo.toml') }}
+          restore-keys: ${{ runner.os }}-cargo-
+
+      - name: Create Rust build for linux
+        if: ${{ !inputs.fake }}
+        env:
+          # Only run tests for native compile
+          FLAGS: ${{ matrix.architecture == 'x86_64' && '-i -r -t' || '-i -r' }}
+        run: docker run --rm -v `pwd`:/io $DOCKER_IMAGE bash -c "cd /io && bash tools/rust_build.sh $FLAGS -g $TARGET -f \"$FEATURES\""
+
+      - name: Create Rust build for linux (FAKE)
+        if: ${{ inputs.fake }}
+        run: |
+          mkdir -p rust/target/${{ env.TARGET }}/release/
+          echo linux >rust/target/${{ env.TARGET }}/release/libopendp.so
+
+      - name: Upload .so
+        uses: actions/upload-artifact@v3
+        with:
+          name: libs
+          # Unnecessary '*' in path will cause artifact to include parent directories starting at that element.
+          path: rust/target*/${{ env.TARGET }}/release/libopendp.so
+
+  python-bindings:
+    needs: credential-check
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.channel }}
+
+      - name: Cache Rust dependencies
+        if: ${{ !inputs.fake }}
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin
+            ~/.cargo/registry/index
+            ~/.cargo/registry/cache
+            ~/.cargo/git/db
+            ~/Library/Caches/gmp-mpfr-sys
+            rust/target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('rust/Cargo.toml') }}
+          restore-keys: ${{ runner.os }}-cargo-
+
+      - name: Create Python bindings
+        if: ${{ !inputs.fake }}
+        run: bash tools/rust_build.sh -r -f "$FEATURES,bindings-python"
+
+      - name: Upload bindings
+        uses: actions/upload-artifact@v3
+        with:
+          name: python_bindings
+          path: python/src/opendp/
```

Comment:
Yeah it just felt cleaner to have this as part of a separate job, rather than as a side effect of one of the lib builds. This lets you disable any platforms, but still get the bindings, which was helpful in testing.

I'll try `cargo check` though.

Reply:
`cargo check` seems to do the trick.